src/components/NotFound/NotFoundWrapper.vue

```diff
@@ -8,7 +8,7 @@
 		</p>
 
 		<div class="tw-flex tw-flex-col tw-text-center tw-my-5 tw-gap-y-2 tw-text-subhead">
-			<router-link to="/">
+			<router-link :to="redirectToMyKivaHomepage ? '/mykiva' : '/'">
```

Comment:
The use of the composable could be simplified through all components by instead providing a method that returns the URL.